kolibri/plugins/facility/assets/src/views/DataPage/index.vue

```diff
@@ -218,7 +218,11 @@
       LearnMoreModal,
       KDateRange,
     },
-    mixins: [commonCoreStrings, KResponsiveWindowMixin],
+    mixins: [commonCoreStrings],
+    setup() {
+      const { windowIsLarge, windowIsSmall } = useKResponsiveWindow();
+      return { windowIsLarge, windowIsSmall };
```

Comment:
I don't see `windowIsLarge` referenced in this file, but I do see `windowIsMedium` referenced, so this should have `windowIsMedium` rather than `windowIsLarge` here and in the line above.

Reply:
Sorry @rtibbles , fixed now!